### DISCUSSION TITLE
try updating plex from command update failed

### DISCUSSION THREAD

**Discussion by platini76** [tteck/Proxmox#2494] [Help Needed]
[ERROR] in line 72: exit code 0: while executing command bash -c "$(wget -qO - https://raw.githubusercontent.com/mrworf/plexupdate/master/extras/installer.sh)"

this is the error... can you please help me?

**Comment by tteck** [comment]
To update Plex, use the command `apt update && apt upgrade`
You're trying to install [plexupdate](https://github.com/mrworf/plexupdate)

**Comment by platini76** [comment]
this is the reply:
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Calculating upgrade... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
You have new mail in /var/mail/root

but it won't upgrade

**Reply by tteck** [reply]
> All packages are up to date.

**Comment by platini76** [comment]
ah ok now yes... thanks!
